src/transformers/dependency_versions_table.py

```diff
@@ -91,7 +91,7 @@
     "tiktoken": "tiktoken",
     "timm": "timm<=1.0.11",
     "tokenizers": "tokenizers>=0.21,<0.22",
-    "torch": "torch>=2.0",
+    "torch": "torch>=2.1",
```

Comment:
Can we specify `torch>=2.1.1`

Reply:
I see. Given that most people are likely using newer torch versions, such specific pin (to the patch version) isn't really necessary IMO.

IIRC, when we need to pin such patch version(s), the reasons were earlier patch version(s) have serious issues and make several models not useable. One example I remembered was audio models failed at some point.

Reply:
I will merge as it is. If we find such pin is necesssary, we can update in the future.